tools/driver/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ if(${LIBSWIFT_BUILD_MODE} MATCHES "BOOTSTRAPPING.*")
   )
   target_link_libraries(swift-frontend-bootstrapping1
                         PRIVATE
+                          swiftCore-bootstrapping0
```

Comment:
Why is this needed?
`libswift-bootstrapping1` already depends on `swiftCore-bootstrapping0` (https://github.com/apple/swift/blob/669e3f34a6839af68e3bb43aff8d19c5be20eac1/libswift/CMakeLists.txt#L37)

Reply:
The users of the library need to specify their link dependencies.  That results in not only the build graph tracking that dependency but actually ensures that the library search path is properly updated to locate the library.  Without the dependency here, we would fail to find the import library for swiftCore resulting in a build failure.